docker/vep/Dockerfile.GRCh37

```diff
@@ -0,0 +1,36 @@
+FROM konradjk/vep85_loftee:1.0.3
+
+ENV VEP_VERSION 85
+ENV VEP_DIR /vep_bin
+ENV REFERENCE_GENOME grch37
+
+RUN echo "APT::Acquire::Retries \"5\";" > /etc/apt/apt.conf.d/80-retries && \
+    mkdir -p /usr/share/keyrings/ && \
+    apt-get update -y && \
+    apt-get install -y build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libreadline-dev libffi-dev libsqlite3-dev wget libbz2-dev && \
+    apt-get install -y curl gnupg software-properties-common && \
+    wget https://www.python.org/ftp/python/3.8.0/Python-3.8.0.tgz && \
+    tar -xf Python-3.8.0.tgz && \
+    cd Python-3.8.0 && \
+    ./configure --enable-optimizations && \
+    make -j 8 && \
+    make altinstall
+
+RUN mv /vep $VEP_DIR
+
+WORKDIR $VEP_DIR/ensembl-tools-release-$VEP_VERSION/scripts/variant_effect_predictor/
+
+RUN sed -i 's/master/main/g' INSTALL.pl
+
+RUN perl INSTALL.pl -a a
+
+WORKDIR $VEP_DIR
+
+RUN echo "#!/bin/bash" > /vep
+RUN echo "export PERL5LIB=$VEP_DIR/loftee:\$PERL5LIB" >> /vep
+RUN echo "perl $VEP_DIR/ensembl-tools-release-$VEP_VERSION/scripts/variant_effect_predictor/variant_effect_predictor.pl \"\$@\"" >> /vep
+RUN chmod +x /vep
```

Comment:
Now that we're happy with this we should collapse the RUN lines into one line. Docker seems to really struggle with images with lots of small layers.